datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -282,21 +282,22 @@ Projection: t1.t1_id, __scalar_sq_1.SUM(t2.t2_int) AS t2_sum
 --------Filter: SUM(t2.t2_int) < Int64(3)
 ----------Aggregate: groupBy=[[t2.t2_id]], aggr=[[SUM(CAST(t2.t2_int AS Int64))]]
 ------------TableScan: t2 projection=[t2_id, t2_int]
-physical_plan
+Physical_plan
```

Comment:
`HashJoinExec` prefers to choose the child having `FilterExec` as the build side now.